.github/workflows/ci-unix-static-sanitized.yml

```diff
@@ -0,0 +1,75 @@
+name: CI Unix Static Sanitized
+on: [ push, pull_request ]
+
+permissions:
+  contents: read
+
+jobs:
+  build-static-sanitized:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [ undefined ] # TODO(yguyon): Add address, memory, thread
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@959f1116cf9f1ae42fff8ec1a4aaae6d4a0e348b # v2.0.1
+        with:
+          # CMake version 3.17 is required to build libwebp (which libsharpyuv is part of) on macOS.
+          cmake-version: '3.17.x'
+      - uses: ilammy/setup-nasm@13cbeb366c45c4379d3478cdcbadd8295feb5028 # v1.5.1
+        with:
+          version: 2.15.05
+      - uses: seanmiddleditch/gha-setup-ninja@8b297075da4cd2a5f1fd21fe011b499edf06e9d2 # v4
+      - run: pip install meson
+
+      - name: Cache external dependencies
+        id: cache-ext
+        uses: actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319 # v4.0.1
+        with:
+          path: ext
+          key: ${{ runner.os }}-${{ matrix.sanitizer }}-${{ hashFiles('ext/*.cmd') }}
+
+      - name: Build aom
+        if: steps.cache-ext.outputs.cache-hit != 'true'
+        working-directory: ./ext
+        run: CC=clang CXX=clang++ CFLAGS=-fsanitize=${{ matrix.sanitizer }} CXXFLAGS=-fsanitize=${{ matrix.sanitizer }} LDFLAGS=-fsanitize=${{ matrix.sanitizer }} ./aom.cmd
```

Comment:
Optional: We can set environment variables using `env`. Please see if this works:

```
      run: ./aom.cmd
      env:
        CC: clang
        CXX: clang++
        CFLAGS: -fsanitize=${{ matrix.sanitizer }}
        CXXFLAGS: -fsanitize=${{ matrix.sanitizer }}
        LDFLAGS: -fsanitize=${{ matrix.sanitizer }}
```

It is also possible to set environment variables for multiple commands.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2060